doc/contribute/guidelines.rst

```diff
@@ -111,7 +111,7 @@ We recommend using the following copyright notice:
 
 .. code-block:: C
 
-    Copyright The Zephyr Project Contributors
+    SPDX-FileCopyrightText: Copyright The Zephyr Project Contributors
```

Comment:
would be very helpful if some context and references are added here, something along the lines of what KDE is doing for example https://community.kde.org/Guidelines_and_HOWTOs/Licensing